containers/products-list/__mocks__/products.js

```diff
@@ -37,19 +37,19 @@ export const products = [
                   nodes: [
                     {
                       largeUrl:
-                        'https://res.cloudinary.com/hn8kfpevp/image/upload/v1/spree/products/39/large/solidus_tshirt.jpg'
+                        'https://res.cloudinary.com/hn8kfpevp/image/upload/v1/spree/products/39/large/solidus_tshirt.jpg',
                     },
                     {
                       largeUrl:
-                        'https://res.cloudinary.com/hn8kfpevp/image/upload/v1/spree/products/40/large/solidus_tshirt_back.jpg'
-                    }
-                  ]
-                }
-              }
-            }
-          ]
-        }
-      }
-    }
-  }
+                        'https://res.cloudinary.com/hn8kfpevp/image/upload/v1/spree/products/40/large/solidus_tshirt_back.jpg',
+                    },
+                  ],
+                },
+              },
+            },
+          ],
+        },
+      },
+    },
+  },
```

Comment:
I'm aware of the usefulness of this formatting rule, but personally, I think that all these commas are stressful 👀 (or maybe it's my OCD? 😂)
It's just an opinion but I think changing the [`Trailing Commas`](https://prettier.io/docs/en/options.html#trailing-commas) to `none` isn't a big problem:
https://github.com/nebulab/solidus_nextjs_frontend/blob/d0d59123a01ab9256774b1a8a569400466e2aea2/.prettierrc#L2

CC @dinohamzic @AlessioRocco for an opinion

Reply:
Agree, I dislike those commas too. @vassalloandrea what do you think?

Reply:
I usually use them, but is not really important for me 👍  I'll rebase it soon